enhancements/console/quick-starts.md

```diff
@@ -0,0 +1,154 @@
+6. All Quick Starts will be listed available in a separate page that will be accessible from Help Menu.
+   ![help-menu](https://raw.githubusercontent.com/jhadvig/images/master/help-menu.png)
+
+
+QuickStarts CR for [Explore Serverless](https://marvelapp.com/236ge4ig/screen/69908905):
+```
+apiVersion: console.openshift.io/v1
+kind: QuickStart
+metadata:
+  name: explore-serverless
+spec:
+  version: 4.7
+  displayName: Explore Serverless
+  duration: 10
+  description: Install the Serverless Operator to enable containers, microservices and functions to run "serverless"
+  prerequisites: Release requirements if any Install X number of resources.
+  introduction: Redhat OpenShift Serverless is a service based on the open source Knative project. It provides ...
+  tasks:
+    - title: Install Serverless Operator
+      description: The OperatorHub is where you can find a catalog of available Operators to install on your cluster ...
+      review: 
+        instructions: Make sure the Serverless Operator was successfully installed ...
+        taskHelp: Try walking through the steps again to properly install the Serverless Operator
+      recapitulation:
+        success: You've just installed the Serverless Operator! Next, we'll install the required CR's for this Operator to run.
+        failed: Check your work to make sure that the Serverless Operator is properly installed
+    - title: Create knative-serving API
+      description: The first CR we'll create is knative-serving ...
+      review:
+        instructions: Make sure the knative-serving API was successfully installed ...
+        taskHelp: Try walking through the steps again to properly create the instance of knative-serving
+      recapitulation:
+        success: You've just created an instance of knative-serving! Next, we'll create an instance of knative-eventing
+        failed: Check your work to make sure that the instance of knative-serving is properly created
+    - title: create knative-eventing API
+      description: The second CR we'll create is knative-eventing ...
+      review:
+        instructions: Make sure the knative-eventing API was successfully installed ...
+        taskHelp: Try walking through the steps again to properly create the instance of knative-eventing
+      recapitulation:
+        success: You've just created an instance of knative-eventing!
+        failed: Check your work to make sure that the instance of knative-eventing is properly created
+  conclusion: Your Serverless Operator is ready! If you want to learn how to deploy a serverless application, take the Serverless Application tour.
+  nextQuickStart: serverless-application
+```
+
+#### Quick Start icon
+
+Each Quick Start CR can contain an icon that is specified in the `spec.icon` field. This field is either an URL of the Package Manifest that contains icon, or a base64 encoded image that are used as Quick Start's icon
```

Comment:
I don't think we should ask quick start authors to build a URL to a package manifest icon. That's error prone and is really an internal implementation detail that could change over time.

Reply:
Wasn't sure with it either but that's it it the outcome of the conversation I had vit @rohitkrai03 and @christianvogt.
Do you propose to cover just the base64 encoded image ?

Reply:
If we want to support this, I'd suggest a specific API for referencing an operator icon by package manifest name.